src/App.js

```diff
@@ -12,11 +14,11 @@ class App extends Component {
   render() {
     return (
       < Router >
-        <Route exact path="/">
-          <div className="App">
-            <Home />
-          </div>
+        <Header />
+        <Route>
```

Comment:
Will this work when you add `/foor` to the URL?
```suggestion
        <Route path="/">
```

Reply:
it worked